homeWork.playground/Contents.swift

```diff
@@ -0,0 +1,43 @@
+var toDoList = [1,10,22,44,5,13,19,23,99,9]
+toDoList.reverse()
+print(toDoList)
+toDoList.randomElement()
```

Comment:
great job @Noura098  but we needed to sort the array in random way (hint search for a method that do that )